test/functional/parallel/parallel-simulators-e2e-specs.js

```diff
@@ -23,7 +23,8 @@ async function resetMapping (mapping) {
   mapping.clear();
 }
 
-describe('XCUITestDriver - parallel Simulators', function () {
+// skip tests in Travis, because they're unstable due to slowness
+describe('XCUITestDriver - parallel Simulators @skip-ci', function () {
```

Comment:
so this is the new approach for skipping tests on CI. how about the current one? does it still work?

Reply:
The way it was being done did not work because each `describe` block is it's own context, and needs to be skipped individually. But in Travis we invoke the tests with mocha, and an exclusion of anything with `@skip-ci` in it (https://github.com/appium/appium-xcuitest-driver/blob/master/.travis.yml#L37). So this is easier.

Reply:
ok, thanks for the explanation